tests/command.bats

```diff
@@ -1342,3 +1342,51 @@ EOF
 
   unstub docker
 }
+
+@test "Load image from archive" {
+  export BUILDKITE_PLUGIN_DOCKER_LOAD='test-image.tar'
+  unset BUILDKITE_PLUGIN_DOCKER_IMAGE
+
+  stub docker \
+    "load -i test-image.tar : echo loaded docker image"
+
+  run "$PWD"/hooks/command
+
+  assert_success
+  assert_output --partial "loaded docker image"
+
+  unstub docker
+}
+
+@test "Load image from gzipped archive" {
+  export BUILDKITE_PLUGIN_DOCKER_LOAD='test-image.tar.gz'
+  unset BUILDKITE_PLUGIN_DOCKER_IMAGE
+
+  stub docker \
+    "load -i test-image.tar.gz : echo loaded docker image"
+
+  run "$PWD"/hooks/command
+
+  assert_success
+  assert_output --partial "loaded docker image"
+
+  unstub docker
+}
```

Comment:
These two tests are failing because now the `run` part of the plugin is unconditional. I believe they should be removed anyways.